BasiliskII/src/Windows/posix_emu.cpp

```diff
@@ -1123,3 +1123,28 @@ int my_write( int fd, const void *buffer, unsigned int count )
 	D(bug("write(%ld,%08x,%ld) = %d\n", fd, buffer, count, result));
 	return result;
 }
+
+FILETIME getFileTime(time_t time) {
+	FILETIME ft;
+	unsigned long long result = 11644473600LL;
+	result += time;
+	result *= 10000000LL;
+	ft.dwHighDateTime = (result >> 32);
+	ft.dwLowDateTime = (result & 0xFFFFFFFF);
+	return ft;
+}
+
+int my_utime( const char *path, struct my_utimbuf * my_times )
+{
+	auto tpath = tstr(path);

```

Comment:
It looks like you use `auto` in several other places in this file. I simply copied the usage pattern from one of the other places. I'll be happy to remove it, but wanted to confirm.